inc/command/class-command.php

```diff
@@ -475,6 +490,8 @@ protected function bootstrap( InputInterface $input, OutputInterface $output ) :
 
 		if ( ! in_array( $subsubcommand, $configs, true ) ) {
 			$output->writeln( sprintf( '<error>Could not find the target configuration set generator for "%s".</error>', $subsubcommand ) );
+			$output->writeln( sprintf( 'Available configuration sets are: %s.', implode( ', ', $configs ) ) );
+
```

Comment:
Not sure what these extra line spaces are before return statements, is this intentional?